tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -71,7 +71,24 @@ def generate_repo_url(list, repo_url, commit_id):
 
 
 def generate_dependencies(list, package_name, version):
-    if (package_name != 'Microsoft.AI.MachineLearning'):
+    if (package_name == 'Microsoft.AI.MachineLearning'):
+        list.append('<dependencies>')
+
+        # Support .Net Core
+        list.append('<group targetFramework="NETCOREAPP">')
+        list.append('<dependency id="Microsoft.Windows.SDK.NET"' + ' version="10.0.18362.3-preview"/>')
```

Comment:
>18362 [](start = 83, length = 5)

How often would this need to get updated?

Reply:
It is updated when either: 
1) we make deliberate changes to the api and require new SDK functionality. The current and only one available is 18362 - aka 19H1.
2) the preview NuGet is promoted to an official release.